app/src/utils/stateRestore.tsx

```diff
@@ -65,7 +65,9 @@ const STORE_CONVERT: any = {
     let task = { ...store.task, id: generateId('task') };
 
     return {
-      selectedScreen: store.selectedScreen || 'task',
+      selectedScreen: {
+        screen: store.selectedScreen?.screen || 'task',
+      },
```

Comment:
Why is the nested object, needed?

Reply:
I was trying to find a nice way to get the createSlice to apply the types to it's state and I couldn't do it without nesting. I agree this isn't the right solution.